diagnostic_feedback/tests/integration/base_test.py

```diff
@@ -0,0 +1,183 @@
+        if load_immediately:
+            return self.go_to_view("student_view")
+
+    def load_student_view(self, xml_file, params=None, load_immediately=True):
+        """
+        Load student view in workbench
+        """
+        params = params or {}
+        scenario = loader.render_template("xml_templates/student_{}.xml".format(xml_file), params)
+        self.set_scenario_xml(scenario)
+        if load_immediately:
+            return self.go_to_view("student_view")
+
+    def load_studio_view(self, xml_file, params=None):
+        """
+        Load studio view in workbench
+        """
+        params = params or {}
+        scenario = loader.render_template("xml_templates/studio_{}.xml".format(xml_file), params)
+        self.set_scenario_xml(scenario)
+        return self.go_to_view("studio_view")
+
+
+class StudentBaseTest(DiagnosticFeedbackBaseTest):
+
+    def _get_startover_button(self):
+        return self.browser.find_elements_by_xpath("//ul[@role='menu']/li")[3]
+
+    def _get_startover_button_link(self):
+        return self.browser.find_element_by_xpath("//ul[@role='menu']/li/a[@href='#cancel']")
+
+    def _is_any_choice_selected(self):
+        selected = False
+        choices = [c for c in self.browser.find_elements_by_xpath("//input[@type='radio']") if c.is_displayed()]
+
+        for choice in choices:
+            if choice.is_selected():
+                selected = True
+                break
+
+        return selected
+
+    def _get_choice(self, idx):
+        return [c for c in self.browser.find_elements_by_xpath("//input[@type='radio']") if c.is_displayed()][idx]
+
+    def _verify_question(self, question_order, next_btn, back_btn, choice_idx):
+
+        self.assertEqual(self._is_any_choice_selected(), False)
+
+        if question_order > 0:
+            self.assertEqual(back_btn.get_attribute('class'), '')
+        else:
+            self.assertEqual(back_btn.get_attribute('class'), 'disabled')
+
+        self.assertEqual(next_btn.get_attribute('class'), 'disabled')
+
+        choice = self._get_choice(choice_idx)
+        choice.click()
+        self.assertEqual(self._is_any_choice_selected(), True)
+
+        if question_order == 0:
+            self.assertEqual(back_btn.get_attribute('class'), 'disabled')
+        else:
+            self.assertEqual(back_btn.get_attribute('class'), '')
+
+        self.assertEqual(next_btn.get_attribute('class'), '')
+
+    def _verify_startover(self, next_btn, back_btn, next_btn_link, back_btn_link, startover_link):
+        startover_link.click()
+
+        time.sleep(50)
+        self.wait_until_visible(next_btn)
+        self.assertEqual(startover_link.is_displayed(), False)
+
+        self.assertEqual(startover_link.is_displayed(), False)
+        self.assertEqual(back_btn.get_attribute('class'), 'disabled')
+        self.assertEqual(next_btn.get_attribute('class'), 'disabled')
+
+        self._verify_question(0, next_btn, back_btn, 0)
```

Comment:
@jareerahsan Might be worth turning the formal parameters that `_verify_question` expects into keyword arguments. When reading this line in isolation, it is hard to tell what some of the arguments "mean" (esp. the first one and the last one).